pcdsdevices/pseudopos.py

```diff
@@ -746,6 +746,7 @@ class DelayBase(FltMvInterface, PseudoPositioner):
     user_offset = Cpt(NotepadLinkedSignal, ':OphydOffset',
                       notepad_metadata={'record': 'ao', 'default_value': 0.0})
     motor = None
+    limits = None
```

Comment:
So these devices already have limits via [`pcdsdevices.pseudopos.PseudoPositioner`](https://github.com/pcdshub/pcdsdevices/blob/02a0bcefe4101d5dc8c91cd744a6f3ca59c5d1ac/pcdsdevices/pseudopos.py#L114) -> [`ophyd.pseudopos.PseudoPositioner`](https://github.com/bluesky/ophyd/blob/1c2b121e63de747f9d3780f6fc208c8912f89d49/ophyd/pseudopos.py#L659-L662)

By overwriting the property here, we're disabling all of that inherited behavior.

Reply:
```python
In [4]: DelayBase.__mro__
Out[4]: 
(pcdsdevices.pseudopos.DelayBase,
 pcdsdevices.interface.FltMvInterface,
 pcdsdevices.interface.MvInterface,
 pcdsdevices.interface.BaseInterface,
 pcdsdevices.pseudopos.PseudoPositioner,
 ophyd.pseudopos.PseudoPositioner,
 ophyd.device.Device,
 ophyd.device.BlueskyInterface,
 ophyd.positioner.SoftPositioner,
 ophyd.positioner.PositionerBase,
 ophyd.ophydobj.OphydObject,
 object)
```

https://github.com/bluesky/ophyd/blob/1c2b121e63de747f9d3780f6fc208c8912f89d49/ophyd/positioner.py#L284-L286